snactor/loader/group.py

```diff
@@ -0,0 +1,96 @@
+import os
+
+from snactor.loader import create_actor, load
+from snactor.registry import get_registered_actors, must_get_actor
+from snactor.utils import get_chan
+
+
+class UnresolvedDependenciesError(Exception):
+    """
+    UnresolvedDependenciesError is thrown when the dependencies between actors can't be automatically resolved
+    """
+    pass
```

Comment:
If I may, I have a few ideas regarding these home-made exceptions that would make the code more idiomatic.

I'd recommend moving our home-made exceptions to a separate  to an `exceptions.py` file. Then, we can create a base exception type, like `SnactorException`, that our own exceptions would derive from. This would allow the runner, for example, to check if the exception raised is from `snactor`.

IMO this is a good practice in a Python library. Good examples of this are the `requests` [1] and `redis-py` [2] libraries.

[1] https://github.com/requests/requests/blob/master/requests/exceptions.py
[2] https://github.com/andymccurdy/redis-py/blob/master/redis/exceptions.py

Reply:
Fair enough - It's a welcome change for a follow patch - I just don't think it's already needed for this PR